training/tf/tfprocess.py

```diff
@@ -64,12 +64,12 @@ def __init__(self, next_batch):
             tf.nn.softmax_cross_entropy_with_logits(labels=self.y_,
                                                     logits=self.y_conv)
         self.policy_loss = tf.reduce_mean(cross_entropy)
-        tf.summary.scalar('policy_loss', self.policy_loss)
 
         # Loss on value head
+        # Google's paper scales MSE by 1/4 to a [0, 1] range, so do the same to
+        # get comparable values.
         self.mse_loss = \
-            tf.reduce_mean(tf.squared_difference(self.z_, self.z_conv))
-        tf.summary.scalar('mse_loss', self.mse_loss)
+            tf.reduce_mean(tf.squared_difference(self.z_, self.z_conv)) / 4.0
```

Comment:
Changing this here means the loss function that is being minimized will change too - not what we want, I think. You can probably create a self.display_loss that is scaled so the optimizer can keep the right one.